pkg/operator/applyconfigurations/internal/internal.go

```diff
@@ -3,8 +3,8 @@
 package internal
 
 import (
-	"fmt"
-	"sync"
+	fmt "fmt"
+	sync "sync"
```

Comment:
How is this change relevant?

Reply:
@chiragkyal / @bharath-b-rh `make generate` calls [hack/update-clientgen.sh](https://github.com/openshift/cert-manager-operator/blob/master/hack/update-clientgen.sh) to leverage the `vendor/k8s.io/code-generator` to do the codegen, there should be some changes, but I unfortunately wasn't able to find a way to disable it.